orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -101,6 +101,7 @@ func (b *baseOrchestratorBuilder) Build(network *storage.Network, graph *storage
 	vals["control_proxy"] = &mconfig_protos.ControlProxy{LogLevel: protos.LogLevel_INFO}
 	vals["metricsd"] = &mconfig_protos.MetricsD{LogLevel: protos.LogLevel_INFO}
 	vals["state"] = getStateMconfig(net, gatewayID)
+	vals["shared_mconfig"] = &mconfig_protos.SharedMConfig{SentryConfig: getNetworkSentryConfig(&net)}
```

Comment:
Nit: seems like the prevalent styling is "Mconfig" rather than "MConfig". I'm a fan of the latter, but consistency is preferable for small stuff like this